### ISSUE TITLE
Edit Group function on WG/BoF non-functional

### ISSUE THREAD

**Issue by martinthomson** [ietf-tools/datatracker#7660]
### Describe the issue

As chair for https://datatracker.ietf.org/wg/diem/about/ that page shows an "Edit Group" button that leads to a page that has an "Additional URLs" form field.

I entered things into that field and submitted the change.  No change was visible.  Subsequent attempts to "Edit Group" show the form field empty again.

Perhaps there is some sort of approval required for this, but that is not communicated.

I was adding a link to deal with #7624 manually, if that helps.

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by rjsparks** [comment]
That the Additional URLs field is still available on the form is surprising, and is a bug. 

It was what existed in the models before we had `GroupExtResources`, replacing `GroupURL`, and the notion is no longer used in the code.

**Comment by rjsparks** [comment]
To do what you were aiming for, try editing the "Additional Resources" and adding
```
webpage https://datatracker.ietf.org/doc/bofreq-haberman-digital-emblems/ (BoF Description)
```

**Comment by martinthomson** [comment]
Thanks Robert.  For information: I used the Additional URLs field because the "Additional Resources" field is a little intimidating.  That is, it is quite a bit more cognitive load to take on.

**Comment by rjsparks** [comment]
Yes, it is a horrible interface, I agree. And the implementation under it is just as fraught. I thought we had an issue to make it saner, but I'm not finding it quickly. If I don't resurface it soon, I'll make another.
